lib/src/easy_localization_state.dart

```diff
@@ -0,0 +1,118 @@
+import 'dart:developer';
+
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:intl/intl_standalone.dart'
+    if (dart.library.html) 'package:intl/intl_browser.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+import 'translations.dart';
+
+class EasyLocalizationState extends ChangeNotifier {
```

Comment:
we have two state, EasyLocalizationState and _EasyLocalizationState, may be rename like EasyLocalizationNotifier?

Reply:
hmm... may be use BLoC pattern best way?

Reply:
Yea i wanted to see what you are thinking about this. 

For me bloc is an overkill, that is why i removed it. 
For the naming i think i would change EasyLocalizationState to LocaleState. 

Thoughts?

Reply:
about `EasyLocalizationState` i like it , then LocaleState :)

Reply:
EasyLocalizationState does not match the State pattern, it's more like Controller pattern. EasyLocalizationController? LocaleState dirty name